include/Board.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+#include "ChessPiece.h"
+#include "Location.h"
+#include "Pawn.h"
+#include "Rook.h"
+#include "Bishop.h"
+#include "Knight.h"
+#include "Queen.h"
+#include "King.h"
+
+#include <string>
+
+
+class Board
+{
+public:
+	static Board& getInstance(const std::string& boardOrder = "");
```

Comment:
why it's static ?? you don't have to access it from every where,  board should be only declared in the chess or in the main and be passed as parameter where ever it's needed